### PR TITLE
Add additional documentation for the use of custom images on the oled.

### PR DESCRIPTION
**Pull request by redoxcode**
This PR adds additional documentation for the use of custom images on the oled display and also addresses potential screen burn in as mentioned here: https://github.com/Allen-Synthesis/EuroPi/issues/83

### CODE REVIEW

File: software/oled_tips.md [hunk 1]
```diff
@@ -52,3 +52,29 @@ oled.centre_text("this text\nhas been\ncentred")
...
+imgFB = FrameBuffer(imgBA, 32, 32, MONO_HLSB)
+# Finally the small FrameBuffer can be blit anywhere onto the display FrameBuffer
+oled.blit(imgFB,40,0)
+oled.show()
+```
+To generate the bytearray string from an jpg file this tool can be used:
+
+https://github.com/novaspirit/img2bytearray
+
+Make sure your jpg has the right size and you use the same size when you run this tool und in the micropython code.
```
Comment (mjaskula) [inline software/oled_tips.md:74]: [required] 'und' -> 'and'

File: software/oled_tips.md [hunk 2]
```diff
@@ -52,3 +52,29 @@ oled.centre_text("this text\nhas been\ncentred")
...
+Images will be inverted. So a black pixel on the jpg will be bright on the oled
+
+# Burn in
+If a script is left running for a long period of time, it can burn that screen into the oled and leave ghost images.
+To avoid this it is recommended to not leave the EuroPi screen on for very long periods of time with with something static on the display.
+If you write a script consider adding a 'screensaver' function (for example clear the screen if there was no user interaction for an extended time periode).
```
Comment (mjaskula) [inline software/oled_tips.md:80]: [required] typo 'periode'

**Review by mjaskula — CHANGES_REQUESTED** [review]
Hello, thanks for your PR. It's always great to have some documentation updates. If you haven't yet, pleas read the [contributing guide](https://github.com/Allen-Synthesis/EuroPi/blob/main/contributing.md) so that you know what to expect during your PR.

This one looks pretty good to me. I highlighted a couple typos to be corrected, and opened a discussion on the image used in the example. Once we have these resolved we can get this merged. Thanks again for your contribution. I hope that there will be more.

File: software/oled_tips.md [hunk 3]
```diff
@@ -52,3 +52,29 @@ oled.centre_text("this text\nhas been\ncentred")
...
 
 ![imgur](https://i.imgur.com/Elljlt1.jpg)
+
+### Custom images
+
+  Custom images can be displayed using the blit function.
+ ```python
+from europi import *
+# The black and white image needs to be stored as bytearray:
+img=b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\x80\x00\x00~\x00\x00\x00<\x00\x00\x00<\x00\x00\x00<\x00\x00\x00>\x00\x00\x00>\x00\x00>>\x00\x00>>\x00\x00>?\x80\x00\x1d\xff\xff\xc0?\xff\xff\xfc\xff\xff\x80\x7f\xff\xff\x80\x7f?\xff\xff\xfc\x1d\xff\xff\xc0>?\x80\x00>>\x00\x00>>\x00\x00\x00>\x00\x00\x00>\x00\x00\x00<\x00\x00\x00<\x00\x00\x00<\x00\x00\x00~\x00\x00\x00\xff\x80\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```
[the remark this answers is not in the document]
Comment (mjaskula) [reply software/oled_tips.md]: 🤦 Yeah, that's it. I just swapped in the bitmap. Sorry. Changing the size reveals a spaceship. I do think the leaf is 'better' though, so thanks for the change.

**Review by mjaskula — APPROVED** [review]
Looks good to me now. Thanks!

**Comment by roryjamesallen** [comment]
LGTM, thanks for adding to the docs!

**Pull request merged**